### PR TITLE
kt-connect 0.3.5 (new formula)

### PR DESCRIPTION
**Pull request by linfan**
- [ YES ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ YES ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ YES ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ YES ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ YES ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ YES ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
This is a re-created PR of #104344, since the commit message can only be changed by a force push.

### CODE REVIEW

File: Formula/kt-connect.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class KtConnect < Formula
+  desc "Toolkit for integrating with kubernetes dev environment more efficiently"
+  homepage "https://alibaba.github.io/kt-connect"
+  url "https://github.com/alibaba/kt-connect/archive/refs/tags/v0.3.5.tar.gz"
+  sha256 "b1a2f0f71feb7d8c4418f73048c24687d513dc1cabb1f68ffac3be5baa0e3f5a"
+  license "GPL-3.0-or-later"
```
Comment (chenrui333) [inline Formula/kt-connect.rb:6]: let's also add head build support
```suggestion
  license "GPL-3.0-or-later"
  head "https://github.com/alibaba/kt-connect.git", branch: "master"
```

**Review by chenrui333 — DISMISSED** [review]

**Comment by linfan** [comment]
is there anything else need to be fixed ?

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
LGTM, Thanks!

**Comment by linfan** [comment]
can someone have a click on the big green merge button now : ) ?

**Comment by chenrui333** [comment]
@linfan, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**